### PR TITLE
DayCamp Website Updates

### PR DESCRIPTION
**Pull request by kelnera**
These are my updates to the following sections: About DayCamp, About
the OSL, and Schedule.

### CODE REVIEW

**Comment by kelnera** [comment]
@ramereth @Kennric @ElijahCaine My part is done if you all could review and make sure everything is right. There's no staging branch so we should probably triple check before giving +1s.

**Comment by kelnera** [comment]
I only checked the written content and not the code changes, but it all looks good to me. +1

**Comment by kelnera** [comment]
FINALLY remembered how to add conditional questions to ticket orders. Can't figure out where to put the link though, but eventbrite is ready to go live. Link is here if @ElijahCaine could add it in the right spot:

https://www.eventbrite.com/e/devops-daycamp-2016-tickets-26911752777

**Comment by kelnera** [comment]
@ramereth @Kennric This still needs more +1s and we're past the tentative deadline for having this done. Can we get this looked at and merged?

File: devopstheme/templates/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     </div>
     <div class='text'>
         <h1>Devops Daycamp</h1>
-        <h2>October 3rd, 2015<br>
+        <h2>October 11th, 2016<br>
```
Comment (tschuy) [inline devopstheme/templates/index.html]: October 1st, isn't it?

Comment (kelnera) [inline devopstheme/templates/index.html]: Hahaha! Yes it is. This has been a real struggle for us.

File: content/pages/osl.rst [hunk 2]
```diff
@@ -5,18 +5,11 @@ OSU Open Source Lab
...
 Drupal. Together, the OSL’s hosted sites deliver nearly 430 terabytes of
-information to people around the world every month. The most active organization
-of its kind, the OSL offers world-class hosting services, professional software
-development and on-the-ground training for promising students interested in open
-source management and programming. The Open Source Lab is now part of a big
-experiential learning center at Oregon State called the `Center for Applied
-Systems and Software`_ (CASS). CASS provides students hands-on experience in
-testing, software development, networking, devops, and hosting.
-
-.. _Center for Applied Systems and Software: http://cass.oregonstate.edu
```
Comment (ramereth) [inline content/pages/osl.rst]: We need to keep this bit about CASS in the description please.

File: content/pages/schedule.rst [hunk 3]
```diff
@@ -4,34 +4,60 @@ Schedule
...
+Introductory Workshop and Navigating Linux
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First we will:
+
+- Review what DevOps and DevOps Bootcamp are.
+- Cover what DevOps BootCamp will teach you throughout the year.
+- Get you started with your own Linux server that that you can use for hands on
+  demos.
+- Get started using that VM!
```
Comment (ramereth) [inline content/pages/schedule.rst]: You might use "Virtual Machine (VM)" in case people don't know what that means.

**Comment by kelnera** [comment]
I just made changes and then there was some merge conflicts and then I resolved them and then I hit commit accidentally before adding a description. Long story short, I put the CASS description back in with a few wording changes.

**Comment by kelnera** [comment]
@ElijahCaine I know what happened. I changed the

**Comment by kelnera** [comment]
I'm just going to walk away before I hit another wrong button. I'll be at my desk if anyone needs me.

**Comment by ramereth** [comment]
+1

**Comment by Kennric** [comment]
Looks ok to me. +1

**Pull request merged**